File: 27. Remove Element/RemEle.java
class Solution {
    public int removeElement(int[] nums, int val) {
        ArrayList<Integer> numArr = new ArrayList<>();
        int k = nums.length;
        for (int i = 0; i < k; i++) {
            if (nums[i] != val) {
                numArr.add(nums[i]);
            }
        }
        for (int i = 0; i < numArr.size(); i++) {
            nums[i] = numArr.get(i);
        }
        return numArr.size();
    }
}

File: 27. Remove Element/RemEle.cpp
#include <vector>
using namespace std;
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int k = nums.size();
        int count = 0;
        for(int i=0;i<k;i++){
            if(nums[i]!=val){
                nums[count++] = nums[i];
            }
        }
        return count;
    }
};
